addons/stock/models/stock_move.py

```diff
@@ -874,7 +874,7 @@ def _assign_picking(self):
         type (moves should already have them identical). Otherwise, create a new
         picking to assign them to. """
         Picking = self.env['stock.picking']
-        grouped_moves = groupby(sorted(self, key=lambda m: [f.id for f in m._key_assign_picking()]), key=lambda m: [m._key_assign_picking()])
+        grouped_moves = groupby(sorted(self, key=lambda m: m._key_assign_picking()), key=lambda m: [m._key_assign_picking()])
```

Comment:
The list for groupby key is probably not needed.

Reply:
> The list for groupby key is probably not needed.

Most likely yes. Should I go ahead and remove it?